cloud/filestore/src/FileRoutes.ts

```diff
@@ -11,8 +11,9 @@ import * as fileService from './services/FileService';
  * @returns router instance of express router
  */
 export function createRouter(storageEngine: FileStorage) {
+  fileService.createTemporaryStorageFolder();
   const router = Router();
-  router.route('/:filename').post(function(req, res, next) {
+  router.route('/base64/:filename').post(function(req, res, next) {
```

Comment:
Do you guys know the exact format needed for `parseBase64Stream`? Guess we can just test it out once this is merged :+1: